include/boost/geometry/util/normalize_spheroidal_coordinates.hpp

```diff
@@ -0,0 +1,258 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2015, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Menelaos Karavelas, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+#ifndef BOOST_GEOMETRY_UTIL_NORMALIZE_SPHEROIDAL_COORDINATES_HPP
+#define BOOST_GEOMETRY_UTIL_NORMALIZE_SPHEROIDAL_COORDINATES_HPP
+
+#include <boost/assert.hpp>
+
+#include <boost/geometry/core/cs.hpp>
+#include <boost/geometry/util/math.hpp>
+
+
+namespace boost { namespace geometry
+{
+
+namespace math 
+{
+
+#ifndef DOXYGEN_NO_DETAIL
+namespace detail
+{
+
+
+template <typename CoordinateType, typename Units>
+struct constants_on_spheroid
+{
+    static inline CoordinateType period()
+    {
+        static CoordinateType const two_pi
+            = math::pi<CoordinateType>() * CoordinateType(2.0);
+        return two_pi;
+    }
```

Comment:
Boost.Math defines quite a few of them. See
http://www.boost.org/doc/libs/1_58_0/libs/math/doc/html/math_toolkit/constants.html

My proposal would be to have (following the Boost.Math types) `math::half_pi<>` and `math::two_pi<>` along with the existing `math::pi<>`.
I can add them to `util/math.hpp` as part of this PR if we agree.